wandb/data_types.py

```diff
@@ -522,6 +528,76 @@ def captions(audio_list):
             return ['' if c == None else c for c in captions]
 
 
+def isNumpyArray(data):
+    np = util.get_module(
+        "numpy", required="Logging Raw Point cloud data requires numpy")
+    return isinstance(data, np.ndarray)
+
+
+class Object3D(IterableMedia):
+    MAX_3D_COUNT = 20
+    SUPPORTED_TYPES = set(['obj', 'gltf', 'babylon', 'stl'])
+
+    def __init__(self, data, **kwargs):
+        """
+        Accepts a 3D File or numpy array
+        """
```

Comment:
We should say which filetypes are supported here, and what the numpy array should look like.  It's actually supports in IO object that has a read method which would be good to point out.  Maybe we add a name keyword argument for when someone isn't passing in a file, like a BytesIO object.